osquery/database/database.cpp

```diff
@@ -35,6 +35,10 @@ CLI_FLAG(bool,
 FLAG_ALIAS(bool, use_in_memory_database, database_in_memory);
 
 FLAG(bool, disable_database, false, "Disable the persistent RocksDB storage");
+FLAG(bool,
+     top_level_decorators,
```

Comment:
Can you define this in: `./config/parsers/decorators.cpp` where `disable_decorators` lives?

Then here use:
```cpp
DECLARE_bool(top_level_decorators);
```